### ISSUE TITLE
Files upload but don't deploy - maximum file size limit / ndpi file reading issue?

### ISSUE THREAD

**Issue by AndyBlake** [cytomine/Cytomine-core-legacy#1180]
I'm adding ndpi images to the docker setup and whenever I try with a file larger than 5gb they upload but fail to deploy. Nothing is shouting out in the logs but maybe you can help?
![Screenshot 2020-04-01 at 10 00 04](https://user-images.githubusercontent.com/1448122/78119459-6154f080-7400-11ea-9811-3836607bb162.png)
I'm emailing the reporting tar to info@cytomine.be with a link to this issue - thanks in advance for your help

**Comment by geektortoise** [comment]
Hello, 

Regarding your logs, the uploads reached the server but Cytomine was not able to identify the format.

I assume that it is related to this OpenSlide limitation
https://github.com/openslide/openslide/issues/174

**Comment by AndyBlake** [comment]
You are right,  thanks so much for the link - looks like it's fixed in that
openslide pull - any idea how I can get that into cytomine?

**Comment by geektortoise** [comment]
Hello, 

A first test could be to compile custom Docker images to integrate this OpenSlide version.

OpenSlide is active in two components : 
- ims for the image metadata fetching
- iipCyto for the tiling.

The Dockerfile for the iipCyto is here : 
https://github.com/cytomine/Dockerfiles/blob/master/iipCyto/Dockerfile#L26
You can change the line by
``RUN cd /tmp && git clone https://github.com/ntrahearn/openslide && cd /tmp/openslide && autoreconf -i && ./configure && make && make install``

And the same modification for ims 
`https://github.com/cytomine/Dockerfiles/blob/master/ims/Dockerfile#L41`

recompile your Docker images with the `docker build -t` command  (as shown in the Dockerfiles README), restart your Cytomine and it would be ready for a test.

I'm not able to currently make the tests because I don't have one of the concerned image in my test set.

Have a nice day

**Comment by geektortoise** [comment]
Hello, @AndyBlake 

Did you succeed with this operation ?

Have a nice day

**Comment by AndyBlake** [comment]
Hiya
Sorry for the delay in replying - Even though the hotfix looked like it should have worked when I tried it with some recompiled binaries it still doesn't seem to handle the larger files. When I have fixed this I'll be sure to update this with how I managed to integrate this into the system - thanks for your help and guidance.
Kind regards
Andy
